package.json

```diff
@@ -1,80 +1,83 @@
-        "@js-joda/locale": "4.14.1",
-        "@js-joda/timezone": "2.21.1",
-        "@ngx-translate/core": "16.0.3",
-        "@ngx-translate/http-loader": "16.0.0",
-        "@ppwcode/js-ts-oddsandends": "1.4.2",
-        "@types/luxon": "3.4.2",
-        "date-fns": "2.30.0",
-        "file-saver-es": "2.0.5",
-        "luxon": "3.5.0",
-        "rxjs": "7.8.1",
-        "tslib": "2.8.1",
-        "zone.js": "0.15.0"
-    },
-    "devDependencies": {
-        "@angular-devkit/build-angular": "19.0.2",
-        "@angular-eslint/builder": "18.4.2",
-        "@angular-eslint/eslint-plugin": "18.4.2",
-        "@angular-eslint/eslint-plugin-template": "18.4.2",
-        "@angular-eslint/schematics": "18.4.2",
-        "@angular-eslint/template-parser": "18.4.2",
-        "@angular/cli": "19.0.2",
-        "@angular/compiler-cli": "19.0.0",
-        "@types/file-saver-es": "2.0.1",
-        "@types/jasmine": "4.3.5",
-        "@typescript-eslint/eslint-plugin": "^8.16.0",
-        "@typescript-eslint/parser": "^8.16.0",
-        "concurrently": "8.2.1",
-        "cross-env": "7.0.3",
-        "eslint": "^9.15.0",
-        "eslint-plugin-depend": "0.12.0",
-        "eslint-plugin-no-secrets": "2.1.1",
-        "jasmine-core": "4.6.0",
-        "karma": "6.4.2",
-        "karma-chrome-launcher": "3.2.0",
-        "karma-coverage": "2.2.1",
-        "karma-jasmine": "5.1.0",
-        "karma-jasmine-html-reporter": "2.1.0",
-        "karma-junit-reporter": "2.0.1",
-        "ng-packagr": "19.0.1",
-        "prettier": "3.1.1",
-        "prettier-config-standard": "7.0.0",
-        "retire": "5.2.5",
-        "typescript": "5.6.3"
-    },
-    "overrides": {
-        "deep-equal": {
-            "regexp.prototype.flags": "npm:regexp.prototype.flags@1.5.0"
-        }
+  "name": "ppwcode",
+  "version": "0.0.0",
+  "scripts": {
+    "start": "ng serve",
+    "start:remote": "ng serve --host 0.0.0.0",
+    "lint:prettier": "cross-env prettier --check \"**/*.{ts,js,md,html,scss}\"",
+    "format:prettier": "cross-env prettier --write \"**/*.{ts,js,md,html,scss}\"",
+    "lint:lint": "ng lint",
+    "format:lint": "ng lint --fix",
+    "ci:test-libs": "./scripts/ci/test-libs.sh",
+    "ci:lint": "npm run lint:prettier && npm run lint:lint",
+    "ci:build": "ng build ppwcode",
+    "ci:build-libs": "./scripts/ci/build-libs.sh",
+    "retire": "retire --outputformat cyclonedx"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "20.2.0",
+    "@angular/cdk": "20.2.0",
+    "@angular/common": "20.2.0",
+    "@angular/compiler": "20.2.0",
+    "@angular/core": "20.2.0",
+    "@angular/forms": "20.2.0",
+    "@angular/material": "20.2.0",
+    "@angular/platform-browser": "20.2.0",
+    "@angular/platform-browser-dynamic": "20.2.0",
+    "@angular/router": "20.2.0",
+    "@js-joda/core": "5.6.5",
+    "@js-joda/extra": "0.12.3",
+    "@js-joda/locale": "4.15.2",
+    "@js-joda/timezone": "2.22.0",
+    "@ngx-translate/core": "17.0.0",
```

Comment:
This is a major upgrade, check the peer dependencies of the packages because they should also allow `^17.0.0`. Same applies for the http-loader.